docker-compose.ci.yaml

```diff
@@ -5,5 +5,8 @@ services:
   web:
     image: rsr-backend:dev
     command: "true"
+  reports:
```

Comment:
I think we do not use this container for the CI server. Maybe we do not need to start it at all?